### PR TITLE
Update dependencies now that GAX 2.7.0 has been released

### PR DESCRIPTION
**Pull request by jskeet**
Will wait for this to go green, then review it carefully myself, before assigning to Chris. (Won't be today...)

### CODE REVIEW

**Comment by jskeet** [comment]
Okay, it's ready for review. The first commit is the manual part, and the commit message explains what I've attempted to do :)

File: apis/apis.json [hunk 1]
```diff
@@ -226,12 +219,12 @@
     "dependencies": {
       "Google.Cloud.Diagnostics.Common": "project",
       "Google.Cloud.Diagnostics.AspNetCore.Analyzers": "1.0.0-beta02",
-      "Grpc.Core": "1.19.0"
+      "Grpc.Core": "1.20.1"
```
Comment (chrisdunelm) [inline apis/apis.json]: Should references to `Grpc.Core` be `default`, rather than a specific version number?
`Google.Cloud.Firestore` uses `default`.

Comment (jskeet) [reply apis/apis.json]: Good point - this could absolutely be the default. It's fine to be default for any non-beta version. Will fix.

Comment (jskeet) [reply apis/apis.json]: I've added another commit for this; it doesn't change the project files (which is exactly what I expected). Will merge on green.

**Comment by chrisdunelm** [comment]
Still LGTM :)

**Pull request merged**